### ISSUE TITLE
Add Other language translation to resources

### ISSUE THREAD

**Issue by preetkaran20** [SasanLabs/VulnerableApp#420]
As we are converting the application to other locales, so this task is to convert to any of your native language.

other tickets:
1. https://github.com/SasanLabs/VulnerableApp/issues/371
2. https://github.com/SasanLabs/VulnerableApp/issues/416

**Comment by natastro** [comment]
Hello! I'd be happy to provide Spanish translations if needed 😁

**Comment by preetkaran20** [comment]
Spanish is already worked upon in #418

**Comment by dafarias** [comment]
Spanish PR is ready at #423. Let me know if something is up 😄

**Comment by yuhwaa** [comment]
Hi! I'd like to work on the traditional Chinese translation if needed!

**Comment by preetkaran20** [comment]
@yuhwaa yes, we don't have Chinese translation. Please go ahead with adding it. 

Thanks, 
Karan

**Comment by yuhwaa** [comment]
@preetkaran20 I've started working on the translation — should I add a new issue or can I respond to this one? I am quite new to open source and I'd like to make sure I am following the right process. Thank you!

**Comment by preetkaran20** [comment]
@yuhwaa please create a newer issue and add the link of this one in the newer issue.

**Comment by antonsixtenson** [comment]
Hey, I can translate to Swedish if no one else is working on it!

**Comment by preetkaran20** [comment]
@antonsixtenson please go ahead.

thanks

**Comment by SelectBillyFromC** [comment]
I can offer Korean translation!

**Comment by preetkaran20** [comment]
@SelectBillyFromC please go ahead and add the translation. 

Thanks, 
Karan

**Comment by MasterD98** [comment]
I can offer Sinhala translation

**Comment by preetkaran20** [comment]
@MasterD98 sure, please go ahead with the changes

**Comment by incursio-xd** [comment]
Hi @preetkaran20  

I reviewed the existing language files and noticed that some of them have fewer labels compared to the latest messages_en_US.properties file, and there may be minor structural inconsistencies.

I would like to cross-check all language files (including Hindi) against the English file and:
- Add any missing labels
- Remove duplicate keys if present
- Ensure exact key consistency across all translations

Please let me know if I can proceed.

Thanks!

**Comment by Sh4dow-BR** [comment]
Hi @preetkaran20,
Would you be interested in a Portuguese translation?

**Comment by preetkaran20** [comment]
@Sh4dow-BR sure, please go ahead.
